src/wamr/CMakeLists.txt

```diff
@@ -82,7 +88,12 @@ llvm_map_components_to_libnames(
 faasm_private_lib(wamrmodule "${LIB_FILES}")
 target_link_libraries(wamrmodule
         wasm
-        aotclib
         wamrlib
-        ${WAMR_LLVM_LIBRARIES}
         )
+if(WAMR_EXECUTION_MODE STREQUAL "AoT")
```

Comment:
See comment above re. making this an option.

Reply:
Sounds good 👍